doc/sphinxman/source/psiapi.ipynb

```diff
@@ -390,7 +390,7 @@
    ],
    "source": [
     "psi4.set_options({'scf_type': 'df',\n",
-    "                  'freeze_core': 'true'})\n",
+    "                  'freeze_core': True})\n",
```

Comment:
For vexing reasons involving freeze_core taking some string values, string true is probably more correct. I haven't checked lately about whether Py boolean works.

Reply:
ok, it's fine.